### PR TITLE
some minor improvement

### PR DESCRIPTION
**Pull request by xhebox**
I have adapted inferllm on riscv vector 0.7.1. And this is the 1/3 PR to prepare for later PRs.

2nd will change the structure of optimized kernel to allow more flexible kernel impl. And 3rd PR will add real riscv kernel support.

### CODE REVIEW

File: application/alpaca/alpaca.cpp [hunk 1]
```diff
@@ -126,7 +126,7 @@ void sigint_handler(int signo) {
     if (signo == SIGINT) {
         printf("\n");
         printf("%s", running_summary.c_str());
-        _exit(130);
+        exit(130);
```
Comment (xhebox) [inline application/alpaca/alpaca.cpp:129]: This is for gprof, which can not record if we use fast exit `_exit`.

File: CMakeLists.txt [hunk 2]
```diff
@@ -72,6 +73,10 @@ else()
   set(CMAKE_CXX_FLAGS " -O3 -g ${CMAKE_CXX_FLAGS}")
 endif()
 
+if(ENABLE_GPROF)
+  set(CMAKE_CXX_FLAGS " -pg ${CMAKE_CXX_FLAGS}")
```
Comment (chenqy4933) [inline CMakeLists.txt:77]: gprof may not be available in macOS， you can print some warning

**Comment by chenqy4933** [comment]
LGTM

File: CMakeLists.txt (hunk 2 above)
Comment (xhebox) [reply CMakeLists.txt]: Done

**Comment by chenqy4933** [comment]
👍

**Pull request merged**